swiftchat.go

```diff
@@ -34,7 +30,7 @@ type Room struct {
 
 var rooms = make(map[int]Room)
 
-func HandleRoom(w http.ResponseWriter, r *http.Request) {
+func HandleRooms(w http.ResponseWriter, r *http.Request) {
 	switch r.Method {
 	case "POST":
```

Comment:
gorilla provide a different way to handle request method, more convinient way from my perspective
check it out please https://github.com/gorilla/mux